packages/installations/src/util/errors.ts

```diff
@@ -30,6 +32,14 @@ export const enum ErrorCode {
 const ERROR_DESCRIPTION_MAP: { readonly [key in ErrorCode]: string } = {
   [ErrorCode.MISSING_APP_CONFIG_VALUES]:
     'Missing App configuration value: "{$valueName}"',
+  [ErrorCode.INVALID_APP_ID_FORMAT]:
+    'Please set your Application ID. A valid Firebase App ID is required to communicate ' +
+    'with Firebase server APIs: It identifies your application with Firebase.' +
+    'Please refer to https://firebase.google.com/support/privacy/init-options.',
+  [ErrorCode.INVALID_API_KEY_FORMAT]:
+    'Please set a valid API key. A Firebase API key is required to communicate with ' +
```

Comment:
Similarly I would just use `apiKey` to match the name in the config object.

Reply:
done